ID localization for debug application cluster - resource usage monitoring

ariscahyadi opened the pull request:
This PR translates `/docs/tasks/debug-application-cluster/resource-usage-monitoring/` as mentioned in #22296

ariscahyadi commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+
+<!-- body -->
+
+Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#resource-metrics-pipeline) atau _pipeline_ [metrik penuh](#full-metrics-pipeline) untuk mengumpulkan statistik pemantauan.
```
Yes, @sftim. Thank you for your comment.

girikuncoro replied:
@ariscahyadi you haven't addressed this yet
```suggestion
Di Kubernetes, pemantauan aplikasi tidak bergantung pada satu solusi pemantauan saja. Pada klaster baru, kamu bisa menggunakan _pipeline_ [metrik sumber daya](#pipeline-metrik-sumber-dayae) atau _pipeline_ [metrik penuh](#pipeline-metrik-penuh) untuk mengumpulkan statistik pemantauan.
```

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
```
nit: verbs needed
```suggestion
Untuk melakukan penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
```

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+<!-- overview -->
+
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
```
digelar doesn't seem familiar, how about adding parentheses (_deploy_) or change to "dijalankan"?

ariscahyadi replied:
Ok!

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+Untuk penyekalaan aplikasi dan memberikan Service yang handal, kamu perlu
+memahami bagaimana aplikasi berperilaku ketika aplikasi tersebut digelar. Kamu bisa memeriksa
+kinerja aplikasi dalam klaster Kubernetes dengan memeriksa Container,
+[Pod](/id/docs/user-guide/pods), [Service](/id/docs/user-guide/services), dan
```
we don't have these yet
```suggestion
[Pod](/docs/user-guide/pods), [Service](/docs/user-guide/services), dan
```

ariscahyadi replied:
Ok. But, I think we need to create this script `update-user-guide-links.py` to handle redirection "user-guide"

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+## _Pipeline_ Metrik Sumber Daya
+
+_Pipeline_ metrik sumber daya menyediakan sekumpulan metrik terbatas yang terkait dengan
+komponen-komponen klaster seperti _controller_ [Penyekala Pod Horisontal](/id/docs/tasks/run-application/horizontal-pod-autoscale), begitu juga dengan utilitas `kubectl top`.
```
HPA is kubernetes object
and typo on link
```suggestion
komponen-komponen klaster seperti _controller_ [HorizontalPodAutoscaler](/id/docs/tasks/run-application/horizontal-pod-autoscaler), begitu juga dengan utilitas `kubectl top`.
```

ariscahyadi replied:
Ok!

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+API ini disediakan pada `/metrics/resource/v1beta1` pada kubelet yang terautentikasi dan
+porta _read-only_.
+
+## _Pipeline_ Metric Penuh
```
```suggestion
## _Pipeline_ Metrik Penuh
```

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+
+_Pipeline_ metrik penuh memberi kamu akses ke metrik yang lebih banyak. Kubernetes bisa
+menanggapi metrik ini secara otomatis dengan mengubah skala atau mengadaptasi klaster
+berdasarkan kondisi saat ini, dengan menggunakan mekanisme seperti penyekala Pod horisontal.
```
```suggestion
berdasarkan kondisi saat ini, dengan menggunakan mekanisme seperti HorizontalPodAutoscaler.
```

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+`custom.metrics.k8s.io` atau API `external.metrics.k8s.io`.
+ 
+
+[Prometheus](https://prometheus.io), sebuah proyek CNCF, yang dapat secara asli memonitor Kubernetes, Node, dan Prometheus itu sendiri.
```
what does "secara asli" mean?

ariscahyadi replied:
"secara asli" mean "native" :)

girikuncoro commented on content/id/docs/tasks/debug-application-cluster/resource-usage-monitoring.md:
```diff
@@ -0,0 +1,57 @@
+ 
+
+[Prometheus](https://prometheus.io), sebuah proyek CNCF, yang dapat secara asli memonitor Kubernetes, Node, dan Prometheus itu sendiri.
+Proyek _Pipeline_ metrik penuh yang bukan merupakan bagian dari CNCF berada di luar ruang lingkup dari dokumentasi Kubernetes.
```
nit
```suggestion
Proyek _pipeline_ metrik penuh yang bukan merupakan bagian dari CNCF berada di luar ruang lingkup dari dokumentasi Kubernetes.
```

ariscahyadi commented:
> Thanks for localizing this page @ariscahyadi . I've left few comments and found some of links to ID pages that we don't have yet. The rest is nit.

Hi @girikuncoro ... I have already addressed all the comments. Please check!

girikuncoro commented:
Thanks @ariscahyadi !

/lgtm
/approve
